recipe/patch_yaml/openff-interchange-models.yaml

```diff
@@ -0,0 +1,14 @@
+# Pydantic v2 has soft incompatibility with v1, which is what all
+# released versions of our software rely on. This is handled in a
+# microrepo (openff-models) which will get a minor version bump
+# when the new version(s) of things is released. One package
+# (openff-interchange) serves as a fulcrum in our stack, so this
+# is the minimal change that handles it for all of our packages
```

Comment:
A shorter version of this is simply that `openff-models` is breaking things with an upcoming 0.2 release and I don't want my existing builds to be broken as a result. This constraint is in my builds moving forward, and this process is the only way I know of to add them to existing builds.